### PR TITLE
feat: Added sshnpd systemd script and README.md file 

### PR DESCRIPTION
**Pull request by cconstab**
**- What I did**
Added some details around how to have sshnpd run as a systemd service

**- How I did it**
Created a .service file and README.md

**- How to verify it**
Checked everything works on my test rig 9it does)

**- Description for the changelog**
Added sshnpd systemd script and README.md file

### CODE REVIEW

File: systemd/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+This directory contains a systemd unit definitions. It runs a the sshnpd component
+of the ssh! no ports in a GNU screen, which can be attached to (for logging etc.)
+with `screen -r`
+
+## Installation
+
+The `sshnpd.service` files should be placed in `/etc/systemd/system` (as root).
+
+Modify the `sshnpd.service` unit to use the appropriate sender and target
```
Comment (cpswan) [inline systemd/README.md]: May also need to modify the user if not running on a pi

File: systemd/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+The services will then start at the next reboot, or can be started manually
+with:
+
+```bash
+sudo systemctl start sshnpd.service
+```
+
+## Usage
+
+When running there will be three detached screens. `screen -r` will list them:
```
Comment (cpswan) [inline systemd/README.md]: There will only be one, and if there is only one `screen -r` will attach to it directly rather than listing it.

**Review by cpswan — CHANGES_REQUESTED** [review]

File: systemd/README.md (hunk 1 above)
Comment (cconstab) [reply systemd/README.md]: agreed

File: pubspec.lock [hunk 3]
```diff
@@ -7,14 +7,14 @@ packages:
       name: _fe_analyzer_shared
```
Comment (cpswan) [inline pubspec.lock]: pubspec.lock shouldn't be committed (and should be in .gitignore)

**Review by cpswan — CHANGES_REQUESTED** [review]

File: pubspec.lock (hunk 3 above)
Comment (cconstab) [reply pubspec.lock]: removed and added to .gitignore

**Review by cpswan — APPROVED** [review]

**Pull request merged**